src/adhocracy_core/adhocracy_core/scripts/__init__.py

```diff
@@ -173,3 +179,68 @@ def get_sheet_field_for_partial(sheet: ISheet,
     """Get sheet field with resource as last parameter to use with partial."""
     registry = get_current_registry(resource)
     return registry.content.get_sheet_field(resource, sheet, field)
+
+
+def import_fixture(asset: str, root: IPool, registry: Registry,
+                   log_only=False):
+    """Import files in fixture directory defined by :term:`asset` ."""
+    fixture = _asset_to_fixture(asset)
+    imports = _get_import_files(fixture)
+    for import_type, import_file in imports:
+        print('Import type "{}" file "{}"'.format(import_type, import_file))
+        if log_only:
+            continue
+        if import_type == 'groups':
+            _import_groups(root, registry, import_file)
+            # TODO don't use private functions
+        elif import_type == 'users':
+            _import_users(root, registry, import_file)
+        elif import_type == 'resources':
+            import_resources(root, registry, import_file)
```

Comment:
Failes with `TypeError: 'module' object is not callable` as the module `adhocracy_core.scripts.import_resources` is called and not the local function with the same name.